posting/list_test.go

```diff
@@ -973,6 +973,53 @@ func createAndDeleteMultiPartList(t *testing.T, size int) (*List, int) {
 	return ol, commits
 }
 
+func TestLargePlistSplit(t *testing.T) {
+	key := x.DataKey(uuid.New().String(), 1331)
+	ol, err := getNew(key, ps, math.MaxUint64)
+	require.NoError(t, err)
+	b := make([]byte, 30<<20)
+	rand.Read(b)
+	for i := 1; i <= 2; i++ {
+		edge := &pb.DirectedEdge{
+			ValueId: uint64(i),
```

Comment:
@joshua-goldstein this test may not be working as expected. I did read the old PR and this will probably need this change https://github.com/dgraph-io/dgraph/pull/7609/files#diff-133386bdae58410e9e509c3a5de8518c6f0402040483ad0e8d76dcc0499bd9a4R1136 right?

Reply:
Hi @skrdgraph , I'm not sure I follow.  The edge.ValueID field is a uint64 in all of the tests in list_test.go.  It is also a uint64 in list.go, e.g. see [here](https://github.com/dgraph-io/dgraph/blob/0c6071b793a5332f68002b3bae61207c4e15b226/posting/list.go#L408-L423).  

To be clear, codec does not have a RemoveRange method in 21.03.  I believe this was introduced later when the team was integrating roaring bitmaps (which 21.03 does not have).

Reply:
So what threw me off is this from the cherry-pick:

- core code change https://github.com/dgraph-io/dgraph/pull/7609/files#diff-133386bdae58410e9e509c3a5de8518c6f0402040483ad0e8d76dcc0499bd9a4R1136
- and corresponding test https://github.com/dgraph-io/dgraph/pull/7609/files#diff-ac97c5574bd3a6cd4c54907f8ebd5f2b28cf994f9a967b06b8f49ebdd6a0170aR1005 (where splits are being tested)

So this is why I raised the question around `uint64`

Reply:
This is resolved